content/docs/user-guide/experiment-management/checkpoints.md

```diff
@@ -97,13 +97,11 @@ $ dvc stage add --name train \
                 --params seed,lr,weight_decay \
                 --checkpoints model.pt \
                 --plots-no-cache predictions.json \
-                --live dvclive \
+                --plots-no-cache dvclive/scalars \
+                --metrics-no-cache dvclive.json
                 python train.py
 ```
 
-💡 The `--live dvclive` option enables our special logger
-[DVCLive](/doc/dvclive), which helps you register checkpoints from code.
```

Comment:
Does the later example ([under **Registering checkpoints in your code**](https://dvc.org/doc/user-guide/experiment-management/checkpoints#registering-checkpoints-in-your-code)) still work without this?

Reply:
Yes. Checkpoints will work even without the `live` section